app/src/main/java/io/intrepid/contest/screens/contestcreation/reviewcontest/CategoriesListFragment.java

```diff
@@ -0,0 +1,57 @@
+package io.intrepid.contest.screens.contestcreation.reviewcontest;
+
+import android.support.annotation.NonNull;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseFragment;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.base.TextValidatableView;
+
+public class CategoriesListFragment extends BaseFragment<ReviewContestPresenter> implements ReviewContestContract.View, TextValidatableView {
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+        MenuItem nextButton = menu.findItem(R.id.action_next);
+        nextButton.setIcon( android.R.drawable.ic_media_next);
+    }
+
+    @Override
+    protected int getLayoutResourceId() {
+        return R.layout.fragment_contest_categories_rv;
```

Comment:
Maybe you can write the full "recycler_view"? And I can follow that in my views too (currently I don't have any though)